### PR TITLE
Change migration bandwidth consumption charts

### PR DESCRIPTION
**Pull request by avivtur**
Signed-off-by: Aviv Turgeman <aturgema@redhat.com>
## 📝 Description

- changing the design for the migration bandwidth consumption chart
- applying activeNamespace to presented data
- restructing and renaming of MonitoringTab directory (which is now removed and 2 new directories instead MigraionsTab and TopConsumersTab)

[UX Doc](https://docs.google.com/document/d/1VWGGOe1UhmhEqGMCN6kE6zl4-kzmd-orpnDzvh7hG6M/edit#)
[Marvel screenshot](https://marvelapp.com/prototype/65da3ge/screen/88774529)

### Before
migration-graphs-b4

### After

https://user-images.githubusercontent.com/67270715/202235883-9ef319c2-0a0f-4e2a-aa6c-bd15af29c8e2.mp4

### CODE REVIEW

**Comment by avivtur** [comment]
/hold
more changes will be added

**Comment by avivtur** [comment]
/hold cancel
changing the y-axis grid to have no more than 4 lines with equal spaces between them to have more consistent UI display for the graphs

File: src/views/clusteroverview/MigrationsTab/components/BandwidthConsumptionCharts/useMigrationChartsData.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+export const useMigrationChartsData: UseMigrationChartsData = (duration, currentTime, timespan) => {
+  const { t } = useKubevirtTranslation();
+  const [activeNamespace] = useActiveNamespace();
+  const baseQuery = useMemo(() => {
+    const namespacedQuery = activeNamespace !== ALL_NAMESPACES_SESSION_KEY;
+
+    return `(sum_over_time(kubevirt_migrate_vmi_data_processed_bytes${
+      namespacedQuery ? `{namespace='${activeNamespace}'}` : ''
+    }[${duration}]))${namespacedQuery ? ' BY (name, namespace)' : ''}`;
+  }, [activeNamespace, duration]);
```
Comment (metalice) [inline src/views/clusteroverview/MigrationsTab/components/BandwidthConsumptionCharts/useMigrationChartsData.ts:32]: cant this be exported to util function?

**Pull request merged**